src/components/ProgressBar/index.tsx

```diff
@@ -68,10 +71,14 @@ export interface ProgressBarProps {
     additionalInfo?: string;
     /** Content that is displayed whenever status is set to "error" or "success". */
     resultText?: string;
+    /** Indicate whether to display heading above the progress bar */
+    displayHeading?: boolean;
```

Comment:
I think we can just use the label to determine whether to display heading or not.

Reply:
Done.